pkg/batch/subscriber/processor/processors.go

```diff
@@ -24,6 +24,7 @@ import (
 
 const (
 	DomainEventInformerName = "domainEventInformer"
+	SegmentPersisterName    = "segmentPersister"
```

Comment:
Please rename it as follows.
Same for the `values.yaml`.

```suggestion
	SegmentUserPersisterName    = "segmentUserPersister"
```